src/pybind/mgr/dashboard/frontend/src/app/ceph/smb/smb-usersgroups-list/smb-usersgroups-list.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, OnInit, TemplateRef, ViewChild } from '@angular/core';
+import { catchError, switchMap } from 'rxjs/operators';
+import { BehaviorSubject, Observable, of } from 'rxjs';
+
+import _ from 'lodash';
+
+import { ActionLabelsI18n } from '~/app/shared/constants/app.constants';
+import { CdTableAction } from '~/app/shared/models/cd-table-action';
+import { CdTableColumn } from '~/app/shared/models/cd-table-column';
+import { CdTableFetchDataContext } from '~/app/shared/models/cd-table-fetch-data-context';
+import { ListWithDetails } from '~/app/shared/classes/list-with-details.class';
+import { Permission } from '~/app/shared/models/permissions';
+
+import { AuthStorageService } from '~/app/shared/services/auth-storage.service';
+import { SmbService } from '~/app/shared/api/smb.service';
+import { SMBUsersGroups } from '../smb.model';
+
+@Component({
+  selector: 'cd-smb-users-list',
+  templateUrl: './smb-usersgroups-list.component.html',
+  styleUrls: ['./smb-usersgroups-list.component.scss']
+})
+export class SmbUsersgroupsListComponent extends ListWithDetails implements OnInit {
+  @ViewChild('groupsNamesTpl', { static: true })
+  groupsNamesTpl: TemplateRef<any>;
```

Comment:
```suggestion
  groupNamesTpl: TemplateRef<any>;
```

Reply:
Same reasoning as before, following naming convention for this resource as: `usersgroups`

Ref: https://docs.ceph.com/en/latest/mgr/smb/#users-and-groups-resource